src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -74,15 +83,19 @@ export const IframeRuntime: React.FC<IProps> =
     };
   }, [url, authoredState, report]);
 
-  useEffect(() => {
-    if (initialHeight) {
-      setIframeHeight(initialHeight);
-    }
-  }, [initialHeight]);
+  const heightFromSupportedFeatures = ARFromSupportedFeatures && containerWidth ? containerWidth / ARFromSupportedFeatures : 0;
```

Comment:
No need to fix this now, but the other behavior of heightFromSupportedFeatures is that it should be limited to the available visible area. This also means means that the width has to be updated in order to preserve the aspect ratio. 

If the height is sent by the interactive then the width is always `100%` regardless of the aspectRatio either from supported features or the the nativeWidth,nativeHeight from LARA.